Support/The Urgent Incident Process

```diff
@@ -0,0 +1,28 @@
+{{{
+  "title": "The Urgent Incident Process",
+  "date": "07-25-2017",
+  "author": "Justin Lentz",
+  "attachments": [],
+  "contentIsHTML": false,
+  "sticky": true
+}}}
+
+
+### Description (goal/purpose)
+
+What classifies as an urgent incident? The accepted definition at CenturyLink Cloud is any incident that impacts two or more customers or an entire product or service. These incidents qualify because they have the capacity to affect a large amount of functionality for customers. These incidents take first priority with the Customer Care team. We will respond, as per service agreement, throughout the incident to keep customers informed of the current status.
+
+### Roles defined as needed during Urgent Incidents
+
+**Incident Commander** – The Incident Commander role is to assemble the team and work with the CapCom, executives, and engineers to investigate, mitigate, monitor, and ultimately resolve an incident. They make sure restoration of service is always the top priority.
+**CapCom** - This role's title was stolen from NASA's "capsule communicator", a person who is dedicated to communications with the crew of a spaceflight. The Capcom is the heart of the communication with the customers. As engineers work to solve the problem, the Incident Commander coordinates resolution, while the CapCom makes sure the customers stay informed. CapCom also communicates workarounds to current customer issues as they are found by Customer Care, ensuring that the Customer Care team is doing everything they can to mitigate the side effects of urgent incidents.
```

Comment:
Please consider this clarification:

**CAPCOM** - This role's title is inspired by NASA's "Capsule Communicator" position.  NASA's CAPCOM is a person who is dedicated to communications between the crew of a spaceflight and Mission Control.  Our CAPCOM provides the essential role of communication with our customers. As engineers work to solve the problem, the Incident Commander coordinates resolution, while CAPCOM makes sure the customers stay informed. CAPCOM also communicates workarounds to current customer issues so that the Customer Care team can work with customers to mitigate the side effects of urgent incidents.